### PR TITLE
show feedback when WebRTC is not supported by the browser

### PR DESCRIPTION
**Pull request by jancborchardt**
What is the correct error handler? I suppose `on('WebrtcError'` is wrong ;)

Please help @fancycode @LukasReschke :)

### CODE REVIEW

**Comment by fancycode** [comment]
According to https://github.com/andyet/SimpleWebRTC there is a property `capabilities` which is the `webrtcsupport` object from https://github.com/HenrikJoreteg/webrtcsupport.

So you should be able to query all sorts of properties from it.

**Comment by fancycode** [comment]
`support: // boolean whether basic WebRTC support exists` looks like what you want ;-)

**Comment by jancborchardt** [comment]
This is now ready for review :)

**Comment by fancycode** [comment]
lgtm if the undefined variable is fixed

File: js/webrtc.js [hunk 1]
```diff
@@ -118,13 +118,24 @@ var webrtc;
...
-				$('#emptycontent p').hide();
+				message = t('spreed', 'Error while accessing microphone & camera');
+				messageAdditional = error.message || error.name;
 			}
 			$('#emptycontent h2').text(message);
 			$('#emptycontent p').text(messageAdditional);
 		});
 
+		if(!OCA.SpreedMe.webrtc.capabilities.support) {
+			console.log('WebRTC not supported', error);
```
Comment (jancborchardt) [inline js/webrtc.js]: Ah right, just a boolean so no error message. Will just remove it.

Comment (jancborchardt) [inline js/webrtc.js]: Fixed

**Comment by jancborchardt** [comment]
@Ivansss @MorrisJobke @rullzer @LukasReschke please review :)

**Comment by MorrisJobke** [comment]
I tested this and fixed another case in IE11 and Edge.

**Comment by MorrisJobke** [comment]
👍

**Comment by jancborchardt** [comment]
:+1:

**Pull request merged**